lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -164,6 +164,8 @@ const FeatureInfoSection = React.createClass({
         const catalogItemName = (this.props.catalogItem && this.props.catalogItem.name) || '';
         const fullName = (catalogItemName ? (catalogItemName + ' - ') : '') + this.renderDataTitle();
         const reactInfo = getInfoAsReactComponent(this);
+        // Silence lint
+        const ExtraComponent = "";
```

Comment:
You can disable eslint warnings on a line-by-line basis.  Search our code for `eslint-disable-line` for an example.